management_app/schema.sql

```diff
@@ -1,17 +1,78 @@
-DROP TABLE IF EXISTS user;
-DROP TABLE IF EXISTS post;
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS professors_point_info;
+DROP TABLE IF EXISTS courses;
+DROP TABLE IF EXISTS scheduled_teaching;
+DROP TABLE IF EXISTS exceptions;
+DROP TABLE IF EXISTS points_constant;
+DROP TABLE IF EXISTS logs;
 
-CREATE TABLE user (
-  id INTEGER PRIMARY KEY AUTOINCREMENT,
-  username TEXT UNIQUE NOT NULL,
-  password TEXT NOT NULL
+
+CREATE TABLE users (
+  user_id INTEGER PRIMARY KEY AUTOINCREMENT,
+  user_name TEXT NOT NULL,
+  user_email TEXT UNIQUE NOT NULL,
+  user_ucinetid TEXT UNIQUE NOT NULL,
+  user_role TEXT NOT NULL,
+  admin INTEGER NOT NULL -- SQLite does not have a separate Boolean storage class. Instead, Boolean values are stored as integers 0 (false) and 1 (true).
+);
+
+CREATE TABLE professors_point_info (
+  user_id INTEGER PRIMARY KEY,
+  year INTEGER PRIMARY KEY,
```

Comment:
I'm not sure if anyone noticed the same issue, but initializing DB from the current schema did not work for me. For SQLite, if we want to use **Composite Primary Key** in a single table, we need to use the format like below:

```
CREATE TABLE professors_point_info (
  user_id INTEGER NOT NULL,
  year INTEGER NOT NULL,
  previous_balance REAL,
  ending_balance REAL,
  credit_due REAL,
  grad_count REAL,
  grad_students TEXT,
  PRIMARY KEY (user_id, year),
  FOREIGN KEY (user_id) REFERENCES users (user_id)
);

- Reference 1: https://stackoverflow.com/questions/217945/can-i-have-multiple-primary-keys-in-a-single-table)
- Reference 2: https://stackoverflow.com/questions/52343571/can-multiple-columns-count-as-one-combined-primary-key

@yuchesu Can you please double-check from your side when you get a chance? I'm not sure if my command is correct.
```

Reply:
For other tables, there may exist the similar issues.